decidim-pages/config/locales/en.yml

```diff
@@ -1,9 +1,9 @@
 ---
 en:
-  components:
-    page:
-      name: Page
   decidim:
+    features:
+      pages:
+        name: Pages
```

Comment:
Components don't exist anymore. Actually, the pages module should probably be singularized alltogether.